python/flask/vercel.json

```diff
@@ -0,0 +1,14 @@
+{
+  "builds": [
+    {
+      "src": "api/index.py",
+      "use": "@vercel/python"
+    }
+  ],
+  "routes": [
+    {
+      "src": "/(.*)",
+      "dest": "api/index.py"
+    }
+  ]
+}
```

Comment:
Similar to other examples, do we want redirect from index to API route?

Reply:
So to get Flask / Django to work, we had to customize the builder through `vercel.json` - this handles setting up the function to render for the defined routes, in this case the index route.